CONTRIBUTING.md

```diff
@@ -90,8 +90,8 @@ because <additional rationale>.
 
 
 ### When adding a new Localization.lang entry
-Add new `Localization.lang("KEY")` to Java file.
-Tests fail. In the test output a snippet is generated which must be added to the English translation file.
+Add new `Localization.lang("KEY")` to a Java file.
+The tests will fail and in the test output a snippet is generated, which must be added to the English translation file.
```

Comment:
Split into two sentences. -- It is too complex in two sentences.

Reply:
Done